### DISCUSSION TITLE
What was the last dynasty in 🇨🇳?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#796] [Galaxy Trivia] [answered]
**Bonus:** What was the first?

**Comment by SANKARAMDAS** [comment]
The Qing (or Ch'ing) dynasty, also called the Manchu (or Manzu) dynasty, was the last of the imperial dynasties of China, spanning from 1644 to 1911/12

The Xia dynasty(the Xia ruled between 2205 and 1766 BC.) is the first dynasty in traditional Chinese historiography. According to tradition, the Xia dynasty was established by the legendary Yu the Great, after Shun, the last of the Five Emperors, gave the throne to him.
According to the chronology based on the "current text" Bamboo Annals, it ruled between 1989 and 1558 BC.

**Comment by peterbaccaro** [comment, marked as answer]
The last dynasty in China was the **Qing Dynasty**, also known as the Manchu Dynasty. It ruled China from 1644 to 1912.

The first dynasty in China is traditionally considered to be the **Xia Dynasty**, although its existence has been a subject of debate among historians. According to historical records and ancient Chinese texts such as the "Bamboo Annals," the Xia Dynasty is believed to have ruled China from around 2070 BCE to 1600 BCE. Due to limited historical records and the absence of archaeological evidence, the existence and details of the Xia Dynasty remain a topic of scholarly discussion. The lack of substantial evidence has led some historians to view the Xia Dynasty as more of a mythological or semi-historical period.

Following the Xia Dynasty, the **Shang Dynasty** (also known as the Yin Dynasty) emerged and ruled over China from approximately 1600 BCE to 1046 BCE. The Shang Dynasty is **considered the first confirmed dynasty in Chinese history**, as its existence is supported by extensive archaeological findings, including oracle bones inscribed with early Chinese script.
